bunsen/pom.xml

```diff
@@ -83,6 +83,12 @@
         not imposed on the end-user of the Bunsen libraries.-->
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Is this dep required too?

Reply:
It's not needed here based on the explanation given in the other comment (here the version for `logback-core` is derived from its parent `logback-classic` since there is no mention of the version in the `dependencyManagement` section),